index.js

```diff
@@ -0,0 +1,20 @@
+const { MongoClient } = require("mongodb");
+require("dotenv").config();
+
+// Replace the uri string with your MongoDB deployment's connection string.
+const client = new MongoClient(process.env.DB_KEY);
```

Comment:
```suggestion
const db = mongoose.connect(`mongodb+srv://root:${process.env.DB_KEY}@<whatever_url>/<whatever_database_name>?retryWrites=true&w=majority`, {}).then(() => {
    console.log("we have connected to db!); // logic on initial connection
}).catch((error) => {
    console.error(error);   // If there is an error, print it out
});

// Disconnect from connection, for testing purposes. Usually add it
// db.disconnect();
```

Example of using mongoose.